client/src/components/FormPage.js

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react'; 
+import { Form } from 'react-bootstrap'; 
+import  Button  from 'react-bootstrap/Button';
+
+
+const FormPage = () => {
+    const [FirstName, setFirstName] = useState(''); 
+    const [SurName, setSurName] = useState('');
+    const [Email, setEmail] = useState(''); 
+    const [SlackAccount, setSlackAccount] = useState('');
+
+    const handleSubmit = (e) => {
+        e.preventDefault(); // this line prevents the submit button from refreshing the page when clicked. 
+    }
+
+  return (
```

Comment:
need an extra indentation here so it is on the same level as the useState hooks lines.

It would be easie to format the whole file - you can right-click it and select format:
![image](https://user-images.githubusercontent.com/38047481/220446756-4f297e5c-f05f-45c5-a1e5-3dc1b4df2cc9.png)